modules/imgproc/src/color.cpp

```diff
@@ -11050,6 +11050,41 @@ inline bool isFullRange(int code)
 
 } // namespace::
 
+// helper function for dual-plane modes
+void cv::cvtColorTwoPlane( InputArray _ysrc, InputArray _uvsrc, OutputArray _dst, int code, int dcn )
+{
+    // only YUV420 is currently supported
+    switch (code) {
+        case COLOR_YUV2BGR_NV21:  case COLOR_YUV2RGB_NV21:  case COLOR_YUV2BGR_NV12:  case COLOR_YUV2RGB_NV12:
+        case COLOR_YUV2BGRA_NV21: case COLOR_YUV2RGBA_NV21: case COLOR_YUV2BGRA_NV12: case COLOR_YUV2RGBA_NV12:
+            break;
+        default:
+            return;
```

Comment:
> return;

This is an error case. You should use `CV_Error()` here.

Reply:
OK, fixed.